test/benchmark/README.md

```diff
@@ -57,7 +96,7 @@ python report.py
 ```
```

Comment:
@nskool as discussed, it might be good to add report.py as part of the automation as well.

Reply:
Will make this part of a separate CR, with an all-inclusive report.

Reply:
This is now fixed in the current PR